health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AppHelpTutorial.js

```
@@ -0,0 +1,205 @@
+
+  const { isLoading: isSelectedFeatureLoading, data: selectedFeatureConfigs } = Digit.Hooks.useCustomAPIHook(selectedFeatureCriteria);
+
+  const helpContents =
+    selectedFeatureConfigs?.[0]?.filter((ele) => ele?.pages?.includes(pathVar))?.sort((x, y) => x?.order - y?.order) || helpContentList;
+  if (isSelectedFeatureLoading) return <Loader />;
+
+  return (
+    <div className="tutorial-wrapper">
+      <div className="tutorial-row">
+        {isSelectedFeatureLoading && <Loader></Loader>}
+        {helpContents.map((item, index) => (
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)